subsys/bluetooth/shell/bt.c

```diff
@@ -56,6 +56,8 @@ static struct bt_le_oob oob_remote;
 
 #define KEY_STR_LEN 33
 
+#define HCI_CMD_MAX_PARAM 65
```

Comment:
Maybe a code comment above this to explain how the value is chosen? (e.g. a mention of the DHKey command)

Reply:
Comment added